### ISSUE TITLE
Upload not working on Linux

### ISSUE THREAD

**Issue by pfeerick** [platformio/platform-timsp430#12]
tl;dr - It looks like the toolchain needs an update for linux, as it is currently failing on a 64bit linux install.

Just remember that PlatformIO has support for the MSP430s also, so dug my MSGP-ESP430G2 launchpad out, and tried to program a 2553 series chip with a simple blink sketch. The toolchain installed fine, compiled fine, and failed to upload with a 

```
error while loading shared libraries: libusb-0.1.so.4: cannot open shared object file: No such file or directory
```

I then tried installing Energia (the *Energia 1.8.7E21 (10/17/2018)* release), and it worked fine out of the box. I noticed that the uploader, mspdebug had changed version though, from 0.22 to 0.24, so deleted the tool from `.platformio/packages/tool-mspdebug/`and copied just the mspdebug binary into it's place, and it's working fine now.

**Comment by pfeerick** [comment]
Binary used : 
[mspdebug.tar.gz](https://github.com/platformio/platform-timsp430/files/3484170/mspdebug.tar.gz)
